docs/personalization/api_reference/tracking_api.md

```diff
@@ -355,24 +355,10 @@ The first one is executed when a recommendation is shown to the user.
 The second is called when a recommendation is clicked or otherwise accepted.
 Sending Rendered events causes as many requests as recommendations to be displayed, a Clickrecommended event is usually sent only once (when a user clicks on a specific recommendation item).
```

Comment:
Why not "`clickRecommended`" has in the example below?
```suggestion
Sending Rendered events causes as many requests as recommendations to be displayed, a `clickRecommended` event is usually sent only once (when a user clicks on a specific recommendation item).
```

It's probably its label name instead of its property name. Could be "Click recommended" in this case, a bit like "Clicked recommended" from its summary https://github.com/ibexa/personalization/blob/v4.6.14/src/lib/Value/Performance/Summary/Event.php#L19

Or all-caps "CLICKRECOMMENDED" like in https://doc.ibexa.co/projects/userguide/en/master/personalization/event_types/